src/coreclr/jit/llvmlssa.cpp

```diff
@@ -920,14 +1245,68 @@ class ShadowStackAllocator
             }
         }
 
+        void PrintDomTree()
+        {
+            static const int STEP = 2;
+
+            class DomTreePrinter final : public DomTreeVisitor<DomTreePrinter>
+            {
+            public:
+                int Depth = 0;
+
+                DomTreePrinter(Compiler* compiler) : DomTreeVisitor<DomTreePrinter>(compiler)
+                {
+                }
+
+                void PreOrderVisit(BasicBlock* block)
+                {
+                    for (size_t i = 0; i < Depth; i++)
+                    {
+                        printf(" ");
+                    }
+                    printf(FMT_BB "\n", block->bbNum);
+                    Depth += STEP;
+                }
+
+                void PostOrderVisit(BasicBlock* block)
+                {
+                    Depth -= STEP;
+                }
+            };
+
+            printf("Doms:\n");
+            DomTreePrinter visitor(m_compiler);
+            visitor.Depth = 1;
+            visitor.WalkTree(m_compiler->m_domTree);
+        }
+
+        void PrintCandidateBitSet(BitVec bitSet, bool isFull = false, bool isEmpty = false)
+        {
+            printf("{ ");
+            if (isFull)
+            {
+                printf("all ");
+            }
+            else if (!isEmpty)
+            {
+                unsigned lclVarIndex;
+                BitVecOps::Iter iter(&m_candidateBitSetTraits, bitSet);
+                while (iter.NextElem(&lclVarIndex))
+                {
+                    printf("V%02u ", m_compiler->lvaTrackedIndexToLclNum(lclVarIndex));
+                }
+            }
+            printf("}");
+        }
+
         void PrintCurrentLiveCandidates()
         {
             if (!m_anyCandidates)
             {
                 return;
             }
 
-            printf("Liveness: { ");
+            // printf("Liveness: { ");
```

Comment:
Do you want to keep this, or it goes with issue 3053?

Reply:
> goes with issue 3053?

Indeed. It's just noise as is: `Liveness { }` for all occurrences.